downstream/modules/builder/con-why-ee.adoc

```diff
@@ -0,0 +1,13 @@
+[id="con-why-ee"]
+
+= Why use {ExecEnvName}?
+
+With {ExecEnvName}, {PlatformName} has transitioned to a distributed architecture by separating the control plane from the execution plane. Keeping automation execution independent of the control plane results in faster development cycles and improves scalability, reliability, and portability across environments. {PlatformName} also includes access to Ansible content tools, making it easy to build and manage {ExecEnvName}.
+
+In addition to speed, portability and flexibility, {ExecEnvName} provide the following benefits:
```

Comment:
```suggestion
In addition to speed, portability, and flexibility, {ExecEnvName} provides the following benefits:
```

Reply:
{ExecEnvName} is plural so provide is correct in this case. Other change complete.